app/assets/javascripts/config.js.erb

```diff
@@ -0,0 +1,4 @@
+window.hypothesis = {
```

Comment:
I'm concerned about a top-level variable named `hypothesis` in the browser, especially if we start embedding the hypothesis JS in our page. Can we make sure this is scoped one level further? i.e.:

```javascript
window.greencommons = {
  hypothesis: {
    // ...
  }
}
```

Reply:
It'd be good to put future JS "config" stuff here as well.

Reply:
You are right. Great suggestion.